tasks/generate_bdi_data.py

```diff
@@ -0,0 +1,312 @@
+import os
+import math
+from cumulusci.tasks.salesforce import BaseSalesforceApiTask
+from cumulusci.tasks.bulkdata import LoadData
+from cumulusci.core.utils import ordered_yaml_load
+from cumulusci.utils import convert_to_snake_case, temporary_dir
+from cumulusci.core.config import TaskConfig
+from datetime import date
+from datetime import timedelta
+from sqlalchemy import create_engine
+from sqlalchemy import Column
+from sqlalchemy import MetaData
+from sqlalchemy import Integer
+from sqlalchemy import Table
+from sqlalchemy import Unicode
+from sqlalchemy.ext.automap import automap_base
+from sqlalchemy.orm import create_session
+
+START_DATE = date(2019, 1, 1)
+
+
+class GenerateBDIData(BaseSalesforceApiTask):
+
+    task_docs = """
+    Use the `num_records` option to specify how many records to generate.
+    Use the `mappings` option to specify a mapping file.
+    """
+
+    task_options = {
+        "num_records": {
+            "description": "how many records to generate",
```

Comment:
more like how many records of each variation, right?

Reply:
Clarified:

https://github.com/SalesforceFoundation/Cumulus/pull/4247/commits/b8c77eea40072c84958d57ddb6084d4730d5c7aa#diff-c6088a9879682c1af25379e1729b8258L31